secboot/tpm_test.go

```diff
@@ -189,6 +192,16 @@ func (s *secbootTPMSuite) TestSeal(c *C) {
 	})
 	defer stubRestore()
 
+	modelRestore := secboot.MockSbAddSnapModelProfile(func(profile *sb.PCRProtectionProfile, params *sb.SnapModelProfileParams) error {
```

Comment:
maybe add something here that counts if/how-often MockSbAddSnapModelProfile() was called so that the test fails if it is not called at all?

Reply:
yes

Reply:
Added this now